lang/cs/Org.Apache.REEF.IMRU/OnREEF/Driver/IMRUConstants.cs

```diff
@@ -25,5 +25,6 @@ internal static class IMRUConstants
         internal const int TreeFanout = 2;
         internal const string MapTaskPrefix = "IMRUMap";
         internal const string UpdateTaskName = "IMRUMaster";
+        internal const string MasterContextId = "MasterRootContext";
```

Comment:
This is use only for master.....thats why....Mapper context Ids are based on partition Ids.